.github/workflows/testing.yml

```diff
@@ -195,3 +197,179 @@ jobs:
 
       - name: Check workability of benchmark tests
         run: make bench-deps bench DURATION=1x COUNT=1
+
+  testing_mac_os:
+    # We want to run on external PRs, but not on our own internal
+    # PRs as they'll be run by the push to the branch.
+    #
+    # The main trick is described here:
+    # https://github.com/Dart-Code/Dart-Code/pull/2375
+    if: (github.event_name == 'push') ||
+      (github.event_name == 'pull_request' &&
+        github.event.pull_request.head.repo.full_name != github.repository) ||
+      (github.event_name == 'workflow_dispatch')
+
+    strategy:
+      fail-fast: false
+      matrix:
+        golang:
+          - 1.13
+        runs-on:
+          - macos-10.15
+          - macos-11.0
```

Comment:
Here we can limit to one macos version (a latest one?)/

Reply:
Moreover, 10.15 is going to EOL in GitHub Actions. And `macos-12` (not `macos-12.0`!) appears now. Please, catch recent updates from https://github.com/tarantool/smtp/commits/master/.github/workflows/testing.yml.

Reply:
According to tarantool.io tarantool supports 11, 12 versions of macOS. Changed according to that information.